### PR TITLE
Contributing docs improvement

### LINKED ISSUE #1302: improving Contributing guide
I saw some good description related to the issues and pull request is missing . I will try to improve this .

### PR DESCRIPTION
**Pull request by dans77777**
Screenshot (162)
fixes #1302

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,10 +12,20 @@ The main project fork lives here in Github:
...
+
+* [https://github.com/G-Research/armada/tree/gh-pages](https://github.com/G-Research/armada/tree/gh-pages)
+
 If you want to brainstorm a potential new feature, hop on over to the Discussions page, listed [below](#discussions).
 
 ## Contributing Guide
 
+### Setup
+
+setup everything you’ll need to get started running and developing Armada:
```
Comment (kannon92) [inline CONTRIBUTING.md:25]:
```suggestion
Setup everything you’ll need to get started running and developing Armada:
```

**Review by kannon92 — CHANGES_REQUESTED** [review]
small suggestion you can apply but LGTM overal

File: CONTRIBUTING.md (hunk 1 above)
Comment (dans77777) [reply CONTRIBUTING.md]: changes are done. pls check once. @kannon92

**Review by kannon92 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,10 +12,20 @@ The main project fork lives here in Github:
 
 * [https://github.com/G-Research/armada](https://github.com/G-Research/armada)
 
+To work with [Armada website](https://armadaproject.io/), please checkout this branch:
```
Comment (jayofdoom) [inline CONTRIBUTING.md:15]: This isn't universally true, for some docs, they need to be updated in master branch (such as this one). I'd suggest linking directly to https://github.com/G-Research/armada/blob/gh-pages/README.txt and https://github.com/G-Research/armada/blob/master/docs/docs-readme.md to help explain the relationship.

**Review by jayofdoom — APPROVED** [review]
My comment can be addressed in this PR or in a follow up, but we should make sure those existing docs get linked into here at some point.

**Pull request merged**